aten/src/ATen/native/PointwiseOps.h

```diff
@@ -11,10 +11,11 @@ struct TensorIterator;
 namespace native {
 
 using pointwise_fn = void (*)(TensorIterator&, Scalar scalar);
+using pointwise_fn_with_beta = void (*)(TensorIterator&, Scalar scalar, Scalar beta);
```

Comment:
lmk if you think the name should be more general. Right now smooth_l1_backwards looks like the only point wise op here that takes in two scalar params